Source/WebCore/platform/graphics/transforms/TransformOperations.h

```diff
@@ -35,90 +36,91 @@ struct BlendingContext;
 class TransformOperations {
     WTF_MAKE_FAST_ALLOCATED;
 public:
+    using const_iterator = Vector<Ref<TransformOperation>>::const_iterator;
+    using const_reverse_iterator = Vector<Ref<TransformOperation>>::const_reverse_iterator;
+    using value_type = Vector<Ref<TransformOperation>>::value_type;
+
     TransformOperations() = default;
-    WEBCORE_EXPORT explicit TransformOperations(Vector<RefPtr<TransformOperation>>&&);
-    
-    bool operator==(const TransformOperations& o) const;
-    
-    void apply(const FloatSize& size, TransformationMatrix& matrix) const { apply(0, size, matrix); }
-    void apply(unsigned start, const FloatSize& size, TransformationMatrix& matrix) const
+
+    explicit TransformOperations(Ref<TransformOperation>&&);
+    WEBCORE_EXPORT explicit TransformOperations(Vector<Ref<TransformOperation>>&&);
+
+    bool operator==(const TransformOperations&) const;
+
+    TransformOperations clone() const
+    {
+        return TransformOperations { m_operations.map([](const auto& op) { return op->clone(); }) };
```

Comment:
Can we move these multi-line function bodies out of the class? If they can’t be in the `.cpp` file, at least we could put them below the class, so it’s easier to see the class’s interface.